Qt4_LimaVideo.py

```diff
@@ -42,31 +48,38 @@
 import time
 import logging
 import gevent
+import struct
 import numpy as np
+
 from PyQt4 import QtGui
-from Lima import Core
+
+try:
+    from Lima import Core
+except:
+    pass
+
+try:
+    import PyTango
```

Comment:
Ok.  I had initially prepared a separate module Qt4_TangoLimaVideo.py but as the decoder code and many other things were the same I decided to put them together in the same module.  
You are right that Tango and Lima are completely independent, they are, as the module shows, two different access methods.
 I tested here (in ALBA) with direct Lima library or with the Tango Server and both seem to work fine. 

But in fact, I have not a strong position in favour or having one or two separate modules.  If you prefer to have them separate it will also be fine with me.  I decided for a common module as, eventually, the number of decoding functions will grow with time, and then there will be an advantage in having one single module.   If you can give it a try as it is fine, if you prefer to have separate modules I can maybe think in a way to extract the decoding code into a separate python module that both HardwareObjects will share.